src/main/kotlin/uk/gov/dluhc/printapi/mapper/CertificateMapper.kt

```diff
@@ -21,6 +23,7 @@ abstract class CertificateMapper {
 
     @Mapping(target = "vacNumber", expression = "java( idFactory.vacNumber() )")
     @Mapping(source = "ero.englishContactDetails.name", target = "issuingAuthority")
+    @Mapping(expression = "java( welshIssuingAuthorityName(message, ero.getWelshContactDetails()) )", target = "issuingAuthorityCy")
```

Comment:
Hi @NathanRussellValtech ,
I think we can simplify the mapping as below and let MapStruct deal with the null values:

<pre>
    @Mapping(source = "ero.welshContactDetails.name", target = "issuingAuthorityCy")
    abstract fun toCertificate(
</pre>

I also checked mapping of the other WelshContactDetails and we don't require certificate language to be Welsh when mapping, so I assumed we should be good if we neglect it here as well.

Here is where we map the contact details when we receive the `SendApplicationToPrintMessage` sqs message and persist the certificate and the print request:

<pre>
  @Mapping(target = "eroEnglish", expression = "java( toEnglishContactDetails(ero) )")
  <b>  @Mapping(target = "eroWelsh", expression = "java( toWelshContactDetails(message, ero) )")</b>
    abstract fun toPrintRequest(
        message: SendApplicationToPrintMessage,
        ero: EroDto,
    ): PrintRequest

    protected fun initialStatus(): List<PrintRequestStatus> {
        val now = Instant.now(clock)
        return listOf(
            PrintRequestStatus(
                status = Status.PENDING_ASSIGNMENT_TO_BATCH,
                dateCreated = now,
                eventDateTime = now
            )
        )
    }

    protected fun toEnglishContactDetails(ero: EroDto): ElectoralRegistrationOffice? =
        electoralRegistrationOfficeMapper
            .toElectoralRegistrationOffice(ero.englishContactDetails, CertificateLanguage.EN)

    protected fun toWelshContactDetails(
        message: SendApplicationToPrintMessage,
        ero: EroDto
    ): ElectoralRegistrationOffice? {
        val shouldPopulateFromEnglishContactDetails =
            message.certificateLanguage == CertificateLanguage.CY && ero.welshContactDetails == null

        if (shouldPopulateFromEnglishContactDetails) {
            return electoralRegistrationOfficeMapper
                .toElectoralRegistrationOffice(ero.englishContactDetails, CertificateLanguage.EN)
        }

    <b>    return electoralRegistrationOfficeMapper
            .toElectoralRegistrationOffice(ero.welshContactDetails, CertificateLanguage.CY)</b>
    }

</pre>